core/include/prometheus/registry.h

```diff
@@ -36,6 +36,20 @@ namespace prometheus {
 /// a data race.
 class Registry : public Collectable {
  public:
+  /// \brief How to deal with repeatedly added family names for a type
+  enum class InsertBehavior {
+    /// \brief Create new family object and append
```

Comment:
Can we add another option, which should be the default: throw whenever trying to add a family with a name that already exists?

Reply:
donw*e**